code/randomized_algorithms/src/randomized_quick_sort/QuickSort.cpp

```diff
@@ -0,0 +1,43 @@
+#include <stdio>
+#include <algorithm>
+using namespace std;
+int size;
+void swap(int *a,int *b){
+	int t = *a; *a = *b; *b = t;
+}
+void print(int *a,int n){
+	int i=0;
+	for(;i<n;i++){printf("%d ",a[i]);}
```

Comment:
```for``` loop contents should be below the loop. Braces are not required for a single statement inside loop

Reply:
Sure. I'll make changes.
Thanks  :)